### PR TITLE
Calendar/postgres

### PR DESCRIPTION
**Pull request by goodtimefamilyband**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -13,8 +13,16 @@ install:
   - source config/settings.test
   - export PYTHONPATH=.
 
+services:
+  - postgresql
```
Comment (alanhdu) [inline .travis.yml:17]: Can we specify we should use `postgresql 9.3` (that's what we run on deployment). Right now, travis is using `9.4`.

File: config/bootstrap.sh [hunk 2]
```diff
@@ -19,6 +19,7 @@ install sqlite3
 install python3
 install python3-dev
 install python3-pip
-install sqlite3
+#install sqlite3
+install postgresql-server-dev-all
```
Comment (alanhdu) [inline config/bootstrap.sh]: Could you doublecheck the provisioning code here? I couldn't get it to run out-of-the-box. (Needed to install `postgresql-9.3`, `libpq-dev`).

At the end, we should be able to run:

``` bash
$ vagrant destroy
$ vagrant up
```

and have things work out of the box.

File: config/settings.dev [hunk 3]
```diff
@@ -7,3 +7,12 @@ export SECRET_KEY=abc123   # don't share production secret
...
 
 export FACEBOOK_ACCESS_TOKEN="752849378144749|2f88b632e830f764d9038f59fdde10db"
+
+#database settings
+#todo: any benefit to using PGHOST, PGPORT, etc environment vars?
+#see http://www.postgresql.org/docs/9.3/static/libpq-envars.html
+export CAL_DB_HOST='localhost'
+export CAL_DB_PORT='5432'
+export CAL_DB_USER='adi_calendar'
+export CAL_DB_PASS='bzYcqT4k' #this should be better hidden
```
Comment (alanhdu) [inline config/settings.dev:17]: This is fine for development. On deployment, we'll have a separate `settings.deploy` that won't be committed to the Github repo.

**Comment by alanhdu** [comment]
Mostly looks good. Could you rebase onto master (get rid of the merge commit)? 

LGTM w/ the rebase and the comments.

File: config/settings.dev (hunk 3 above)
Comment (sunyang713) [inline config/settings.dev]: welp, the secret key is on the internet

**Comment by goodtimefamilyband** [comment]
I've tested this by destroying / up'ing the box, seems to work OK.  Added some SQL commands to bootstrap.sh to properly set up the DB as well

**Pull request merged**